llabake/Hello-Books
# feature(protect route): using Higher order component

#### What does this PR do?
It uses a higher order component to protect route that require authentication
#### Description of Task to be completed?
add authentication higher order component
change font awesome CDN
use HOC in for authentication in protected routes
delete protect route component

#### How should this be manually tested?
clone the repo
cd into the project directory
run `npm install` to install all the dependency
run `npm run start-dev` to start the back end server
run `npm run dev` to start the front end sever
navigate to protected route without signin up 
you should be redirected to the home page
#### What are the relevant pivotal tracker stories?
157527623

#### Any background context you want to add?
N/A

#### Screenshots

N/A
